CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone/ContextualPopupDecorator` and `sandstone/Dropdown` to have sibling DOM node as alternative to findDOMNode API which will be removed in React 19
 - `sandstone/PageViews` to hide dot page indicator when there is only one page
+- `sandstone/ThemeDecorator` to pass `rootId` to `spotlight/SpotlightRootDecorator`
```

Comment:
I'm not sure we should add this changelog or not.. 🤔 I'll get back to you after discussing this with members.

Reply:
We think it's okay not to have this change log. Because it's merely using it.